bagisto/bagisto
# Added improvemnet #10168

## Issue Reference
Added improvement #10168

## Description
Added configuration setting for handling censoring product reviewer names on the product page.

Config path: configuration -> catalog -> products -> Review -> Censoring Reviewer Name

## How To Test This?
You can test with enable or disable the configuration and then check on the product view page in the review section. 

## Branch Selection
